modules/UI/toolbars/Toolbar.js

```diff
@@ -703,25 +725,25 @@ Toolbar = {
     },
 
     /**
-     * Initialise main toolbar buttons.
+     * Initialise toolbar buttons.
      */
-    _initMainToolbarButtons() {
-        interfaceConfig.MAIN_TOOLBAR_BUTTONS.forEach((value, index) => {
+    _initToolbarButtons() {
+        interfaceConfig.TOOLBAR_BUTTONS.forEach((value, index) => {
+            let place = getToolbarButtonPlace(value);
+
             if (value && value in defaultToolbarButtons) {
                 let button = defaultToolbarButtons[value];
-                this._addMainToolbarButton(
+                this._addToolbarButton(
                     button,
-                    (index === 0),
-                    (index === interfaceConfig.MAIN_TOOLBAR_BUTTONS.length -1),
-                    (interfaceConfig.MAIN_TOOLBAR_SPLITTER_INDEX !== undefined
-                        && index
-                            === interfaceConfig.MAIN_TOOLBAR_SPLITTER_INDEX));
+                    place,
+                    (interfaceConfig.MAIN_TOOLBAR_SPLITTER_INDEX &&
```

Comment:
It's correct. I just prefer to check if configItem exist. Then try to compare it with something. 

I've reseted my changes for `MAIN_TOOLBAR_SPLITTER`